### ISSUE TITLE
Dependency upgrade: Jinja2, suppor version 3

### ISSUE THREAD

**Issue by JohnPreston** [koxudaxi/datamodel-code-generator#463]
Hello,
Would it be possible to evaluate compatibility in using Jinja2 v3+ ? Installed that in my venv and not encountering any issues with 3.0.1 
```bash
Jinja2==3.0.1
datamodel-code-generator==0.11.8
```
Many thanks,

**Comment by koxudaxi** [comment]
@JohnPreston 
Thank you for creating this issue.
OK, I will update the dependency.

**Comment by koxudaxi** [comment]
@JohnPreston 
```
    jinja2>=2.10.1,<4.0
```
I have changed the dependency and released a new version as  0.11.9
